### DISCUSSION TITLE
No model is loaded! Select one in the Model tab.

### DISCUSSION THREAD

**Discussion by SLB13** [oobabooga/text-generation-webui#1206] [Q&A] [answered]
I installed everything correctly, and then when it came to the model I used the "download-model" installer and selected one of the model. I can see in the models folder my model (facebook_opt-6.7b), but when I run the "start-webui" and type something in the website I get the message in the terminal: "No model is loaded! Select one in the Model tab."

**Comment by Lillmackish** [comment]
I'm running into the same issue, but with two separate models installed. I've tripple checked that I've done it correctly, but the start-webui.bat isn't giving me the option of choosing a model.

**Reply by SLB13** [reply]
It's from the website that you select the model, there are tabs in the top of the page. ZombieAlex's comment helped out with this.

**Reply by Nantris** [reply]
I get the error even after selecting the model from the dropdown. Any advice?

**Reply by jasongyorog** [reply]
I used the Model tab to download "EleutherAI/gpt-j-6b" from Hugging Face. After it had downloaded, I clicked the refresh button on the model list so that it would show the newly downloaded model. I selected "EleutherAI_gpt-j-6b" from the drop-down list and clicked the Chat tab to try it. I got ValueError("No model is loaded! Select one in the Model tab.")

The problem was that I had not clicked the "Load" button after selecting the model. The message gives the impression that choosing it in the dropdown would be enough. It may help for the ValueError message to say "Select one and then Load it in the Model tab" instead.

To make this selection persistent, I added this new line to the bottom of CMD_FLAGS.txt ( which gets read by one_click.py when that script is called from start_windows.bat ). Note the underscore, rather than the slash:

`--model EleutherAI_gpt-j-6b`

**Comment by ZombieAlex** [comment, marked as answer]
In the web page, go to the "Model" tab at the top and select your model from the "Model" drop down in that tab.  Or you can edit your start-webui.bat to include the `--model <your model here>` or `--model-menu` to choose your model at startup.

**Reply by SLB13** [reply]
Thank you. Just to ask, is this thing connected to the Internet, or is it just that it's an easy way to get a model up and running on a website? Where's the value I mean, because I found that it's fairly slow (Understandably since it's running on a single (GPU 3070))

**Reply by SparklingWaterEnthusiast** [reply]
I am also curious to know if running it offline is possible.

**Reply by Bob202222** [reply]
Hi, when I open the notepad to edit the startwebui, where should I add the --model-menu?

**Comment by noobbuddy** [comment]
I am getting this error now did I put in wrong place 
```
usage: webui.py [-h] [--update]
webui.py: error: unrecognized arguments: --model TheBloke_wizard-mega-13B-GPTQ

Done!
Press any key to continue . . .
```

**Reply by somewhat69** [reply]
do TheBloke/
